src/exercises/exerciseSequenceTypes.js

```diff
@@ -14,6 +14,8 @@ import MultipleChoiceContext from "./exerciseTypes/multipleChoiceContext/Multipl
 import { MEMORY_TASK } from "./ExerciseTypeConstants";
 
 const NUMBER_OF_BOOKMARKS_TO_PRACTICE = 12;
+const MAX_NUMBER_OF_BOOKMARKS_EX_SESSION = 20;
```

Comment:
do we still need `NUMBER_OF_BOOKMARKS_TO_PRACTICE` if we have the new constant? does that become maybe `MIN_NUMBER...` or can we just remove it?

Reply:
Currently, it's still what we are going to use when assigning exercises if there are too many. I don't think MIN makes sense, as that would imply users could only practice a minimum of 12, and that's not the case. Maybe we can call it PREFERRED_NUMBER_OF_BOOKMARKS_TO_PRACTICE?

Reply:
I see. Maybe call it `DEFAULT`?